docs/api/en/core/Object3D.html

```diff
@@ -206,6 +206,7 @@ <h3>[property:Object userData]</h3>
 		<p>
 			An object that can be used to store custom data about the Object3D. It
 			should not hold references to functions as these will not be cloned.
+			Default is an empty object `{}`.
```

Comment:
Would you also apply the same change to the `userData` field of `Material`, `Texture` and `BufferGeometry`?

Reply:
Done